Makefile.toml

```diff
@@ -157,6 +157,7 @@ done
 [tasks.fetch]
 dependencies = [
   "fetch-sdk",
+  "fetch-toolchain",
```

Comment:
I wonder if we're losing useful granularity with this "fetch" task.  There are a number of places we have granular fetch-* dependencies, and a few places that just fetch everything, and I think most of them wouldn't need the toolchain, for example.

If it were more granular, I think the user wouldn't need to change their Docker config unless they were building a kmod kit...

Reply:
In the kmod kit rework for supporting multiple kernels, I ended up needing the toolchain for variant builds since that's the point in time where we have the information we need about the kernel we're installing in the image.